### DISCUSSION TITLE
Gas estimation errors collecting and distributing rewards after upgrade

### DISCUSSION THREAD

**Discussion by jim-counter** [subquery/subql#960] [Frontier TestNet - Bug Reporting]
I've just updated to `onfinality/subql-coordinator:v0.1.8` and `onfinality/subql-indexer-proxy:v0.1.7`, started re-indexing the projects and am still seeing gas estimation errors in the logs:

`coordinator_service  | 2022-05-03T10:12:36.483Z <transaction> WARN collect and distribute rewards: FAILED : Error: cannot estimate gas; transaction may fail or may require manual gas limit [ See: https://links.ethers.org/v5-errors-UNPREDICTABLE_GAS_LIMIT ] (error={"reason":"processing response error","code":"SERVER_ERROR","body":"{\"jsonrpc\":\"2.0\",\"error\":{\"code\":-32603,\"message\":\"execution fatal: Module { index: 10, error: 0, message: None }\"},\"id\":186}\n","error":{"code":-32603},"requestBody":"{\"method\":\"eth_estimateGas\",\"params\":[{\"type\":\"0x2\",\"maxFeePerGas\":\"0xd09dc300\",\"maxPriorityFeePerGas\":\"0x59682f00\",\"from\":\"0x7a6b4344bbd5e1af749d8d0a61812eee46631e6d\",\"to\":\"0xa500a3ad54223b7e18d7d73da34f2ef06aca77ec\",\"data\":\"0xc731196a00000000000000000000000082aada6f7adedc1b68bdfa40d09581d6bb65ef76\"}],\"id\":186,\"jsonrpc\":\"2.0\"}","requestMethod":"POST","url":"https://sqtn.api.onfinality.io/public"}, tx={"data":"0xc731196a00000000000000000000000082aada6f7adedc1b68bdfa40d09581d6bb65ef76","to":{},"from":"0x7A6b4344Bbd5E1AF749d8D0A61812eeE46631e6d","type":2,"maxFeePerGas":{"type":"BigNumber","hex":"0xd09dc300"},"maxPriorityFeePerGas":{"type":"BigNumber","hex":"0x59682f00"},"nonce":{},"gasLimit":{},"chainId":{}}, code=UNPREDICTABLE_GAS_LIMIT, version=abstract-signer/5.6.0)`

I notice that my era is stuck now too.
![image](https://user-images.githubusercontent.com/11335994/166438813-2570adb9-59ad-4bcd-91fb-e11eff6c96fb.png)

Could be related but I'm unable to announce I'm indexing either.

I can see that the transactions to announce indexing are all queued up in Metamask.
![image](https://user-images.githubusercontent.com/11335994/166438770-f4961bf4-0e02-4c7b-b7ee-11e674987858.png)

**Comment by jim-counter** [comment]
I have now been able to announce indexing. Still seeing the gas estimation errors.

**Comment by jim-counter** [comment]
Please close - the root cause was that my controller account had whittled the DEV balance down to 0.0749. A topup from the faucet has fixed the gas estimation errors.

**Comment by S3VENK** [comment]
I have same problem

**Comment by Hacker-web-Vi** [comment]
I hade the same problem recently
Thanks!
